bin/desi_compute_fiberflat_vs_humidity

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+import sys,os
+import astropy.io.fits as pyfits
+from astropy.table import Table
+import fitsio
+import numpy as np
+import glob
+from scipy.ndimage.filters import median_filter
+from scipy.signal import fftconvolve,correlate
+
+
+import matplotlib.pyplot as plt
+
+from desiutil.log import get_logger
+from desispec.io import findfile,read_fiberflat
+
+log=get_logger()
+
+year=2021
+
+wave=None
+fiberflats_in_bin=None
+bhumids_in_bin=None
+hdulist=None
+
+for spectro in range(10) :
+    cam=f"b{spectro}"
+
+    humidity_table_filename=f"humidity_table_{cam}.csv"
+    if not os.path.isfile(humidity_table_filename) :
+
+        vals=dict()
+        vals["NIGHT"]=[]
+        vals["HUMIDITY"]=[]
+
+        # loop on nights with fiberflat
+        for flat_filename in sorted(glob.glob(f"/global/cfs/cdirs/desi/spectro/redux/daily/calibnight/{year}*/fiberflatnight-{cam}-*.fits")) :
```

Comment:
Thanks for including this script to document how the files are made and give us a starting point for making updates.  We could deal with this later, but eventually we'll need to change the hardcoded `year` to options for a date range (e.g. to make a model after replacing the collimators), and update the hardcoded production path to use `desispec.io.specprod_dir()` (e.g. if we wanted to make an updated cleaner model using a consistent reference production that had used better CCD calibrations).

Reply:
done